server/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "server",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
```

Comment:
We can use nodemon as dev-dependency.

**Why?**
To watch for file changes and auto restart server.

Reply:
Implemented.